### PR TITLE
Predict open spots

### PR DESCRIPTION
**Pull request by kevin-rodri**
I used the getPredictedOpenSpots and getPredictedClosedSpots method to help me make the predictedOpenSpots method (probably should've given it another name). Is that good enough?

### CODE REVIEW

File: src/ParkingAPI.java [hunk 1]
```diff
@@ -51,4 +51,16 @@ public ParkingLot availableParkingLots(ParkingLot one, ParkingLot two) {
 			return null;
 		}
 	}
+	public ParkingLot predictedOpenSpots(ParkingLot one, ParkingLot two) {
+		int lotOne = getPredictedOpenSpots(one) - getPredictedClosedSpots(one);
+		int lotTwo = getPredictedOpenSpots(two) - getPredictedClosedSpots(two);
```
Comment (LittleTealeaf) [inline src/ParkingAPI.java]: Once again, I think you messed up the logic again in the exact same way as last time. I assume that you just want to get the predicted open spots, which should just be `getPredictedOpenSpots()`.

case example: Prediction states that there will be 2 open spots of the 100 total spots. `getPredictedOpenSpots() - getPredictedClosedSpots()` would return `-98`

**Review by LittleTealeaf — CHANGES_REQUESTED** [review]
Looks good, just check up on the logic in the predictedOpenSpots method

File: src/ParkingAPI.java (hunk 1 above)
Comment (kevin-rodri) [reply src/ParkingAPI.java]: Yes, that's what I tried to implement, but I realized that you already have that taken care of.. .

Comment (kevin-rodri) [reply src/ParkingAPI.java]: Should fix it, although I feel as though we may not even need the printlns in availableParkingLots and redictedOpenSpots methods...

**Review by LittleTealeaf — APPROVED** [review]
Looks good to me

**Pull request merged**